src/CheckoutSdk/MetaData/Card/MetaDataCardPayoutsTransactionsType.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Runtime.Serialization;
+
+namespace Checkout.Metadata.Card
+{
+    public enum MetadataCardPayoutsTransactionsType
```

Comment:
How about calling this `CardPayoutsTransactionsType`?

Reply:
👍